core/commands/pin/remotepin.go

```diff
@@ -643,7 +643,11 @@ func getRemotePinService(env cmds.Environment, name string) (*pinclient.Client,
 	if err != nil {
 		return nil, err
 	}
-	return pinclient.NewClient(url, key), nil
+	endpoint, err := normalizeEndpoint(url)
+	if err != nil {
+		return nil, err
+	}
+	return pinclient.NewClient(endpoint, key), nil
 }
 
 func getRemotePinServiceInfo(env cmds.Environment, name string) (url, key string, err error) {
```

Comment:
whether in this PR or the docs one we should rename url to endpoint in this file, we missed a couple

Reply:
Will do in docs one.

Reply:
Actually, I decided to do the remaining part here, to minimize the number of conflicts: e4bc5510649e1f53ace2e0407957ab7c5c35b04c